eml/api/userApi.js

```diff
@@ -127,26 +125,16 @@ export const updateUserPassword = async (user_id, oldPassword, newPassword, toke
   }
 };
 
-export const completeExercise = async (user_id, exercise_id, isComplete, points, token) => {
+export const completeComponent = async (user_id, comp, isComplete, points, token) => {
```

Comment:
What is the role of `isComplete` here? Please add some comments above the functions.

Reply:
isComplete is the bool to set the component to complete in backend. If u look at the exerciseScreen, u can see that if user answers wrong then isComplete is false